operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -221,7 +221,26 @@
             message: '{{ $labels.job }}/%(alert_instance_variable)s has a number of mmap-ed areas close to the limit.' % $._config,
           },
         },
-      ] + [
+      ] + (
+        if $._config.forwarding_enabled then [
+          {
+            alert: $.alertName('DistributorForwardingErrorRate'),
+            expr: |||
+              rate(cortex_distributor_forward_errors_total{}[1m])
+              / on (job, pod)
+              rate(cortex_distributor_forward_requests_total{}[1m])
+              > 0.01
```

Comment:
Shall we use `sum by(%(alert_aggregation_labels)s ...` to compare overall errors vs requests?

Reply:
Also `on (job, pod)` should be based on the variables (e.g. `pod` should be replaced with `alert_instance_variable`)

Reply:
I fixed it, PTAL